.github/workflows/nuclia_sync.yml

```
@@ -1,46 +1,35 @@
 name: Nuclia Sync
 
-on: [push]
+on:
+  push:
+
 jobs:
   sync:
     runs-on: ubuntu-latest
 
     steps:
       - name: Check out code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
-      - name: Set up Python
-        uses: actions/setup-python@v4
+      - name: Set up Python ${{ matrix.python-version }}

```

Where is this declared? There is no `matrix` of Python versions. There is only one. I'd revert this line.

```suggestion
      - name: Set up Python

```